source/beamline/BioXAS/BioXASBeamlineDef.h

```diff
@@ -20,7 +20,8 @@ namespace BioXASBeamlineDef {
 		PseudoM1Motor,
 		PseudoMaskMotor,
 		PseudoMonoMotor,
-		PseudoM2Motor
+		PseudoM2Motor,
+		DetectorStageMotor
```

Comment:
This might not be specific enough (maybe it's not relevant to have it defined here either) because the Main endstation will have 2 detector stage motors and the imaging will have at least 1 (maybe two as well).